app/views/docs/command-line-deployment.phtml

```diff
@@ -45,14 +45,22 @@ The Apprite CLI allows you to create and deploy collections and functions to you
 ✓ Success Deployed Awesome Function ( 621229798628cf5bf712 )</code></pre>
 </div>
 
-<h2><a href="/docs/command-line-deployment#deployCollections" id="deployCollections">Deploying collections</a></h2>
+<h2><a href="/docs/command-line-deployment#deployCollections" id="deployCollections">Deploying Databases and Collections</a></h2>
 
-<p>The Appwrite CLI also helps you migrate your project's collections from a development server to a production server. You can deploy all the collections in your <a href="/docs/command-line-deployment#appwriteJSON">appwrite.json</a> file using:</p>
+<p>The Appwrite CLI also helps you migrate your project's databases and collections from a development server to a production server. You can deploy all the databases and collections in your <a href="/docs/command-line-deployment#appwriteJSON">appwrite.json</a> file using:</p>
 
 <div class="ide margin-bottom" data-lang="bash" data-lang-label="CLI">
     <pre class="line-numbers"><code class="prism language-bash" data-prism>appwrite deploy collections</code></pre>
 </div>
 
+<h2><a href="/docs/command-line-deployment#deployBucket" id="deployBucket">Deploying Storage Buckets</a></h2>
```

Comment:
Let's have one more for teams, and one for functions

Reply:
Done